scalecube-organization/src/main/java/io/scalecube/organization/opearation/ServiceOperation.java

```diff
@@ -194,4 +206,19 @@ protected void checkLastOwner(String userId, Organization organization)
               organization.id()));
     }
   }
+
+  protected Predicate<ApiKey> apiKeyFilterBy(Role role) {
+    return apiKey -> {
+      try {
+        String roleName = apiKey.claims().get("role");
+        if (roleName != null) {
+          Role apiKeyRole = Role.valueOf(roleName);
+          return role.isEqualsOrHigherThan(apiKeyRole);
+        }
+      } catch (Exception ignore) {
+        // no-op
```

Comment:
Maybe let's add at least some logs?

Reply:
no, it's because we use `Enums.valueOf`. it will be better to create a specific method which won't throw exception.